From 3c1bb3dd0fd3f15a7b4f8726675c1b313d5253a3 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Mon, 9 Oct 2023 11:42:01 +0200
Subject: [PATCH 0140/1695] plat-stm32mp1: enable fixed clock in stm32mp15

Enable fixed clock in stm32mp15 that is needed by the platform clock
driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4cfe87db85f762467c51b08778087e32c3b28a2b plat-stm32mp1: enable fixed clock in stm32mp15]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Iea48e1bfbd5ca2df389dce30a128cb66417d0eee
---
 core/arch/arm/plat-stm32mp1/conf.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 9929cc787..e017881a8 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -142,7 +142,7 @@ endif # CFG_STM32MP13
 
 ifeq ($(CFG_STM32MP15),y)
 $(call force,CFG_BOOT_SECONDARY_REQUEST,y)
-$(call force,CFG_DRIVERS_CLK_FIXED,n)
+$(call force,CFG_DRIVERS_CLK_FIXED,y)
 $(call force,CFG_HALT_CORES_ON_PANIC_SGI,15)
 $(call force,CFG_SECONDARY_INIT_CNTFRQ,y)
 $(call force,CFG_STM32MP1_SHARED_RESOURCES,y)
-- 
2.47.2

